### ISSUE TITLE
Test swallows output buffer when run in a separate process

### ISSUE THREAD

**Issue by decadent** [sebastianbergmann/phpunit#1149]
``` php
<?php

class A extends \PHPUnit_Framework_TestCase {

    public function testOne() {
        print '1';
        file_put_contents('aaa1', ob_get_level());
    }

    /**
     * @runInSeparateProcess
     */
    public function testTwo() {
        print '2';
        file_put_contents('aaa2', ob_get_level());
    }

}
```

```
$ phpunit -v a.php
PHPUnit 3.7.31 by Sebastian Bergmann.

.1.

Time: 155 ms, Memory: 33.75Mb

OK (2 tests, 0 assertions)
$ cat aaa1
1
$ cat aaa2
2
```

**Comment by Flimm** [comment]
What version was this released in?

To answer my own question, it was released in v4.1.5.

```
$ git describe --contains 8337d8f93fc451252455120a8e0bd0cc7d5659cf
4.1.5~9
```

Which is odd, because I am running v4.7.6, and I am still experiencing this issue of PHPUnit swallowing output when run in separate processes.

**Comment by DavidSoussan** [comment]
I am experiencing the same issue:  PHPUnit swallowing output when run in separate processes. using version 5.6.3
